octobot_trading/data/portfolio_profitability.py

```diff
@@ -90,7 +90,8 @@ async def _update_profitability(self, force_recompute_origin_portfolio=False):
         try:
             await self.update_portfolio_and_currencies_current_value()
 
-            if not self.origin_portfolio:
+            if self.portfolio_origin_value == 0:
+                # try to update portfolio origin value if it's not known yet
```

Comment:
How can this happen ?

Reply:
When symbol prices are missing at startup